### ISSUE TITLE
Export organization-level code security alerts and push protection data to to CSV

### ISSUE THREAD

**Issue by github-product-roadmap** [github/roadmap#989]
### Summary

We're enhancing user experience and data accessibility with new CSV export functionality on the security overview dashboard and secret scanning metrics page at the organization level. This functionality will enable you to easily download detailed data on alerts from the overview dashboard and data regarding secrets blocked by push protection on the secret scanning metrics page.

### Intended Outcome

This feature empowers you with the tools to perform offline analysis of alerts and and secrets blocked by push protection. By exporting this data to a CSV file, you will gain the ability to:

- conduct in-depth offline analysis
- compare and integrate data with external systems
- archive data for compliance and historical analysis

### How will it work?

You'll find an "Export to CSV" button on both the security overview dashboard and the secret scanning metrics page. Clicking this button will automatically download the underlying data that backs the report, which will reflect the filters currently applied to the respective pages. Downloads are permitted only for repositories where you have the appropriate [permission](https://docs.github.com/enterprise-cloud@latest/code-security/security-overview/about-security-overview#permission-to-view-data-in-security-overview).

Looking forward, we plan to expand this functionality to include API access, enabling automated data retrieval.

**Comment by ankneis** [comment]
This has shipped: https://github.blog/changelog/2024-08-06-csv-exports-for-security-alerts-on-the-organization-level-security-overview-dashboard

Leaving open to track for GHES release.

**Comment by ankneis** [comment]
A new issue will be opened if this ships to GHES in the future.
